From 655e1158d412058ac68618c1e9e2a1e07f2599a5 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sun, 19 Mar 2017 15:18:31 -0400
Subject: [PATCH 07/14] 4.6.1-1 did not fix package license Btrfs-progs is
 explicitly GPL2 and not GPL2+.

eg: SUSE and RedHat examples:

GPL2:
https://build.opensuse.org/package/view_file/openSUSE:Factory/btrfsprogs/btrfsprogs.spec
http://pkgs.fedoraproject.org/cgit/rpms/btrfs-progs.git/tree/btrfs-progs.spec

vs GPL2+:
https://build.opensuse.org/package/view_file/network:utilities/iptraf/iptraf.spec
http://pkgs.fedoraproject.org/cgit/rpms/iptraf-ng.git/tree/iptraf-ng.spec
---
 debian/copyright | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/copyright b/debian/copyright
index 914dd2e6..3ae2af71 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -5,7 +5,7 @@ Files-Excluded: tests/sha224-256.c tests/sha.h tests/sha-private.h
 
 Files: *
 Copyright: 2007-2012 Oracle <http://www.oracle.com/>
-License: GPL-2+
+License: GPL-2
 
 Files: radix-tree.c
 Copyright: 2007-2009 Oracle <http://www.oracle.com/>
-- 
2.11.0

